### PR TITLE
update project text on SECURITY.md

### PR DESCRIPTION
**Pull request by Ranchhand87**
# Description

_text edit on boiler security.md

# Checklist

### Quality
* [ Y] Commit sequence makes sense and have useful messages, see [ref][contrib#git-history].
* [ ] New tests are added and existing tests are updated.
* [ ] Self-reviewed the PR.

### Maintenance
* [ ] Linked an [issue][link-issue] or added the PR to the current sprint of [`ouroboros-network`][project] project.
* [ Y] Added labels.
* [ ] Updated changelog files.
* [ ] The documentation has been properly updated, see [ref][contrib#documentation].

[project]: https://github.com/orgs/IntersectMBO/projects/5/views/1
[link-issue]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword=
[contrib#git-history]: https://github.com/IntersectMBO/ouroboros-network/blob/master/CONTRIBUTING.md#git-history
[contrib#documentation]: https://github.com/IntersectMBO/ouroboros-network/blob/master/CONTRIBUTING.md#documentation

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ as possible.
 
 Please provide a clear and concise description of the vulnerability, including:
 
-* the affected version(s) of Open-Source-Office,
+* the affected version(s) of ouroboros-network,
```
Comment (coot) [inline SECURITY.md:11]: Should it be _the affected version(s) of all packages included in ouroboros-network repository_?

Comment (Ranchhand87) [reply SECURITY.md:11]: Yes, can resend with it saying " the affected version(s) of all packages included in ouroboros-network-repository"

Comment (Ranchhand87) [reply SECURITY.md]: Updated text. @coot

File: SECURITY.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ as possible.
 
 Please provide a clear and concise description of the vulnerability, including:
 
-* the affected version(s) of Open-Source-Office,
+* the affected version(s) of all packages included in ouroboros-network-repository,
```
Comment (coot) [inline SECURITY.md:11]:
```suggestion
* the affected version(s) of all packages included in ouroboros-network repository,
```

**Review by coot — APPROVED** [review]
LGTM, thanks @Ranchhand87

**Pull request merged**